1_fetch/src/fetch_sb_data.R

```diff
@@ -0,0 +1,19 @@
+download_sb_file <- function(sb_id, file_name, out_dir){
```

Comment:
I like that having this wrapper function makes it easy to return the file path that is useful for `format="file"` (as opposed to calling `item_file_download` within the target itself, which probably is not as easy to read anyway).